### PR TITLE
Feat v1.3 update sales page, create rule view UI & mechanics

### PR DESCRIPTION
**Pull request by bali710**

### CODE REVIEW

File: components/scoped/CreateNewRule.js
Comment (immutable-bee) [inline components/scoped/CreateNewRule.js:1]: Update Line 168: 

"setType("standard");"

 to

"setType("STANDARD");"

File: pages/api/pricing-rules/add.js
Comment (immutable-bee) [inline pages/api/pricing-rules/add.js:1]: lgtm

File: pages/api/pricing-rules/update.js
Comment (immutable-bee) [inline pages/api/pricing-rules/update.js:1]: lgtm

File: prisma/schema.prisma
Comment (immutable-bee) [inline prisma/schema.prisma:1]: lgtm

**Review by immutable-bee — CHANGES_REQUESTED** [review]
The "SALE"/"STANDARD" checkbox highlight will switch to "SALE" when clicked but won't switch back to "STANDARD" when clicked again.

File: components/scoped/CreateNewRule.js
Comment (bali710) [reply components/scoped/CreateNewRule.js:1]: I ask Ibrahim for this change He is already taking care of this in another branch

**Pull request merged**